Functions/xASL_bids_BIDSifyASLJSON.m

```diff
@@ -380,7 +345,7 @@
 if isfield(jsonOut,'NumberOfAverages') && (max(jsonOut.NumberOfAverages) > 1)
 	if isfield(studyPar,'TotalAcquiredPairs')
 		if max(jsonOut.NumberOfAverages) ~= studyPar.TotalAcquiredPairs
-			warning('Discrepancy in the number of averages');
+			warning('Discrepancy in the number of averages...');
```

Comment:
We can make the warning more specific:
'Discrepancy in the number of averages from DICOM and TotalAcquiredPairs specified in studyPar.json'

Reply:
4c6babf 👍